scripts/set-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+VERSION=$1
+
+if [ -z "${VERSION}" ] || [ "${VERSION}" == "-h" ] || [ "${VERSION}" == "--help" ]; then
+  echo "set-version.sh <version> [<appVersion>] - Update the chart version and regenerate generated content"
+  echo
+  echo "  <version> The version number to use for the chart."
+  echo "  <appVersion> The app version number to use for the chart (default to latest release)."
+fi
+
+PLUGIN_VERSION=$(gh release list --repo grafana/grafana-k8s-plugin --limit 1 --json name --jq '.[].name')
```

Comment:
Might be worth noting this is a private repo

Reply:
True. This would only be for maintainers, so Grafana Employees.
I want to make an override (didn't finish that part, I guess)
I'll also add a check before hand and a helpful error message.